ios/FluentUI.Demo/FluentUI.Demo/Demos/PopupMenuDemoController_ObjC.m

```diff
@@ -0,0 +1,174 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+#import "PopupMenuDemoController_ObjC.h"
+#import <FluentUI/FluentUI-Swift.h>
+
+@implementation PopupMenuDemoController_ObjC
+
+- (instancetype)init {
+    self = [super init];
+    if (self != nil)
+    {
+        _selectedCityIndex = [[NSIndexPath alloc] initWithIndex:0];
+    }
+    return self;
+}
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    MSFButton *demoButton = [[MSFButton alloc] initWithStyle:MSFButtonStylePrimaryOutline];
+    [demoButton setTitle:@"Show PopupMenu" forState:UIControlStateNormal];
+    [demoButton addTarget:self action:@selector(showPopupMenu) forControlEvents:UIControlEventTouchUpInside];
+    
+    UIStackView *stack = [[UIStackView alloc] initWithArrangedSubviews:@[demoButton]];
+    [stack setTranslatesAutoresizingMaskIntoConstraints:NO];
+    UIView *view = [self view];
+    [view addSubview:stack];
+    [view setBackgroundColor:[MSFColors surfacePrimary]];
+    UILayoutGuide *safeArea = [view safeAreaLayoutGuide];
+    [NSLayoutConstraint activateConstraints:@[
+        [[stack topAnchor] constraintEqualToAnchor:[safeArea topAnchor] constant:10],
+        [[stack leadingAnchor] constraintEqualToAnchor:[safeArea leadingAnchor] constant:10],
+        [[stack trailingAnchor] constraintEqualToAnchor:[safeArea trailingAnchor] constant:-10],
+        [[stack bottomAnchor] constraintEqualToAnchor:[safeArea bottomAnchor] constant:10]
+    ]];
+}
+
+- (void)showPopupMenu {
+    MSFPopupMenuItem *montreal = [[MSFPopupMenuItem alloc] initWithImageName:@"Montreal"
```

Comment:
is it my github or the alignment of this function is weird?

Reply:
It's aligned on the colon, which is how Xcode wanted to format it. In Swift, we usually line up the start of parameters, but I think this is fairly normal for Objective C?